blog/_posts/2021-05-18-argocd-private-clusters.md

```diff
@@ -0,0 +1,414 @@
+---
+layout: post
+title: Bring GitOps to your private Kubernetes clusters with ArgoCD and inlets PRO
```

Comment:
This seems like a descriptive title.

I wonder if there's something that would resonate more with people who are "googling" for a solution or browsing email newsletters and it would make them stop and think: that's what I need to try!

For instance: Learn how to manage apps across private Kubernetes clusters

Then GitOps would be mentioned in the description / intro rather than the title.